Integrations/integration-Carbon_Black_Enterprise_Response.yml

```diff
@@ -1900,5 +1937,58 @@ script:
     - contextPath: CbResponse.Watchlists.CbWatchlistID
       description: Watchlist ID
     description: Retrieve watchlist in Carbon black Response.
+  - name: cb-binary-download
+    arguments:
+    - name: md5
+      required: true
+      default: true
+      description: MD5 File Hash
+    - name: summary
```

Comment:
should be with predefined options of yes and no

Reply:
done